fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -264,6 +264,7 @@ private void handleQuery() {
             } catch (ParsingException parsingException) {
                 throw new AnalysisException(parsingException.getMessage());
             } catch (Exception e) {
+                LOG.warn("failed to parse stmt with new parser", e);
```

Comment:
Adding warn logs here may cause a large number of warn logs to be printed. Any statement not supported by the new Parser will be printed, causing confusion for user